common/djangoapps/config_models/admin.py

```diff
@@ -78,3 +88,108 @@ def revert(self, request, queryset):
                 args=(target.id,),
             )
         )
+
+
+class ShowHistoryFilter(ListFilter):
+    """
+    Admin change view filter to show only the most recent (i.e. the "current") row for each
+    unique key value.
+    """
+    title = 'Status'
```

Comment:
I don't think the Django admin views are normally internationalized, much like log messages or most other ops/dev related strings - these sort of very technical strings are an undue burden on translators and not generally likely to be used much. But I could be wrong. I will double-check tomorrow.